### DISCUSSION TITLE
How to do this function? - Conventions

### DISCUSSION THREAD

**Discussion by zimuyou2** [squidfunk/mkdocs-material#7268] [Q&A] [answered]
![Screenshot_20240615-110559_Via](https://github.com/squidfunk/mkdocs-material/assets/96246963/f1e55935-b09f-4f6f-9563-106a6548c2a7)
![Screenshot_20240615-112048_Chrome](https://github.com/squidfunk/mkdocs-material/assets/96246963/c4755c0a-5336-4071-94f5-fba9f668b4e5)

How is the button on this agreement page made?

Thank you.

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @zimuyou2,
those are https://squidfunk.github.io/mkdocs-material/conventions/ icons and are custom to Material for MkDocs documentation.
I'm not sure what you mean specifically with "button on agreement page", but in general this was asked many times before, closing and locking as duplicate:
- https://github.com/squidfunk/mkdocs-material/discussions/6544
